sidechain/storage/src/test_utils.rs

```diff
@@ -50,28 +54,46 @@ pub fn create_signed_block_with_parenthash(
 	block_number: u64,
 	parent_hash: BlockHash,
 ) -> SignedSidechainBlock {
-	default_block_builder()
+	let header = default_header_builder()
 		.with_parent_hash(parent_hash)
-		.with_number(block_number)
+		.with_block_number(block_number)
+		.build();
+
+	let block_data = default_block_data_builder().build();
+
+	SidechainBlockBuilder::default()
+		.with_header(header)
+		.with_block_data(block_data)
 		.build_signed()
 }
 
 pub fn create_signed_block_with_shard(
 	block_number: u64,
 	shard: ShardIdentifier,
 ) -> SignedSidechainBlock {
-	default_block_builder()
+	let header = default_header_builder()
 		.with_shard(shard)
-		.with_number(block_number)
+		.with_block_number(block_number)
+		.build();
+
+	let block_data = default_block_data_builder().build();
+
+	SidechainBlockBuilder::default()
+		.with_header(header)
+		.with_block_data(block_data)
 		.build_signed()
 }
 
-fn default_block_builder() -> SidechainBlockBuilder {
-	SidechainBlockBuilder::default()
-		.with_signer(ed25519::Pair::from_string("//Alice", None).unwrap())
+fn default_header_builder() -> SidechainHeaderBuilder {
+	SidechainHeaderBuilder::default()
 		.with_parent_hash(H256::random())
-		.with_parentchain_block_hash(H256::random())
-		.with_timestamp(SystemTime::now().duration_since(UNIX_EPOCH).unwrap().as_millis() as u64)
+		.with_block_number(Default::default())
 		.with_shard(default_shard())
-		.with_number(Default::default())
+}
+
+fn default_block_data_builder() -> SidechainBlockDataBuilder {
+	SidechainBlockDataBuilder::default()
+		.with_timestamp(SystemTime::now().duration_since(UNIX_EPOCH).unwrap().as_millis() as u64)
```

Comment:
Also here we could use `itp-time-utils` for the timestamp. 
If this is only used in a `#[cfg(test)]` environment, add it under `[dev-dependencies]`.